### ISSUE TITLE
[Question]: What IPv6 peering address must be used for BGP peering?

### ISSUE THREAD

**Issue by laurentdina** [cilium/cilium#21733]
Hi Cilium community, 
I'm trying to setup a BGP peering from a Node running Cilium v1.12.2 on Ubuntu 22.04 Server to a Cisco router which is acting as a route reflector. I can reach the route reflector from a pod via ICMP but I can not establish BGP peering over IPv6. The entire setup has to run over IPv6.
On the Cisco IOS XR I've used the IPv6 address of the outgoing interface on the Ubuntu node as peering address.
The one configured in the CiliumBGPPeeringPolicy is right, as other router have successfully establish peering to the route reflector.

Is it the correct IPv6 address? Does this even work with IPv6?

Thanks in advance.

**Comment by xiaods** [comment]
even BGP feature is beta, you also can use https://docs.cilium.io/en/v1.12/gettingstarted/bgp/ to verify your concerns.

**Comment by laurentdina** [comment]
> even BGP feature is beta, you also can use https://docs.cilium.io/en/v1.12/gettingstarted/bgp/ to verify your concerns.

Thanks, I've already read the documentation. The peering works now. The problem was due to the multiple network interfaces on the host system. I had to manipulate the addresses and route tables of the Cilium host machine. It's also interesting that the Cilium Host interface uses the same IPv6 address as the first network interface. In my case, I wanted the BGP peering to go over the second one.

I'll close this issue.
